docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -362,6 +362,34 @@ When you are ready to migrate your eZ Publish XmlText content to the eZ Platform
 
 `php -d memory_limit=1536M bin/console ezxmltext:convert-to-richtext --export-dir=ezxmltext-export --export-dir-filter=notice,warning,error --concurrency 4 -v`
 
+**Custom tags**
```

Comment:
this should be purely about custom attributes right?

Reply:
Well, I here also mentioned that custom tag configurations needs to be migrated too.

But we could potentially also add a `**Custom attributes**` line before line 371 (`"If you have configured custom attributes in legacy (...)"`) ?

Reply:
Ok, my thinking is that given custom attributes are across all tags, it should be separate topic, and not tied to custom tags.

Or WDYT?